### ISSUE TITLE
Long Names wrap poorly in Gradebook using Firefox for Mac

### ISSUE THREAD

**Issue by saiena** [drlippman/IMathAS#113]
Using latest Firefox for Mac OS, long names (e.g.: firstname: Priscilla, lastname: Caballero Gonzalez) wrap poorly. See attached screenshot.

![unnamed](https://user-images.githubusercontent.com/458602/35936060-074c2f54-0bf7-11e8-9549-8df17278e5f1.png)

**Comment by drlippman** [comment]
Yup.  I have yet to figure out a good fix for that.  The header-locking script (which is a hobbled together pile of hacks, I'll admit) relies on the browser to do the initial layout.  Not sure how to work around this.
